doc/Programming-guide/Key-platform-features/Data-collection/index.md

```diff
@@ -149,25 +146,28 @@ For the purpose of this guide, we will use schema that is very close to the comm
 }
 ```
 
-In Admin UI as a tenant developer, you can create new log schemas for the application as follows:
+To create a new log schema for your application:
+
+1. Log in to the **Administration** page as a Tenant Developer using the default **devuser** username and **devuser123** password.
```

Comment:
Default **devuser** username and **devuser123** password used in our Sandbox. This guide is not only for Sandbox, thus leave only "Log in to the **Administration** page as a Tenant Developer" or add information that devuser/devuser123 is default Sandbox credentials.